### ISSUE TITLE
Disabling localStorage breaks the homepage

### ISSUE THREAD

**Issue by zgreen** [webpack/webpack.js.org#1247]
When `localStorage` is disabled in the browser, I see the following error on https://webpack.js.org/:

```
Uncaught DOMException: Failed to read the 'localStorage' property from 'Window': Access is denied for this document.
    at t.get (https://webpack.js.org/f6d26b64452ff3cecebf13a6e66bc366.js:2:11186)
    at t.value (https://webpack.js.org/f6d26b64452ff3cecebf13a6e66bc366.js:2:10225)
    at N (https://webpack.js.org/f6d26b64452ff3cecebf13a6e66bc366.js:1:21149)
    at t.render (https://webpack.js.org/f6d26b64452ff3cecebf13a6e66bc366.js:1:21249)
    at L (https://webpack.js.org/f6d26b64452ff3cecebf13a6e66bc366.js:2:6488)
    at A (https://webpack.js.org/f6d26b64452ff3cecebf13a6e66bc366.js:2:5994)
    at D (https://webpack.js.org/f6d26b64452ff3cecebf13a6e66bc366.js:2:7586)
    at C (https://webpack.js.org/f6d26b64452ff3cecebf13a6e66bc366.js:2:3798)
    at E (https://webpack.js.org/f6d26b64452ff3cecebf13a6e66bc366.js:2:3497)
    at Object.R [as render] (https://webpack.js.org/f6d26b64452ff3cecebf13a6e66bc366.js:2:8079)
```

And the homepage looks like this:
<img width="1326" alt="screen shot 2017-05-26 at 8 47 00 am" src="https://cloud.githubusercontent.com/assets/1700355/26495186/8bc1b21c-41f0-11e7-9005-805a4072e71f.png">

Users may choose to disable `localStorage` for security/privacy reasons (as I do). Having said that, it's likely that most visitors will have `localStorage` enabled.

Happy to submit a PR, if you want. Thank you for building this site, the webpack v2 docs are terrific!

**Comment by simon04** [comment]
A PR is very welcome :+1:

**Comment by skipjack** [comment]
I think this is caused by the `Notification` component which uses `localStorage` to track whether users have viewed the notification.

As @simon04 mentioned, a PR would be welcome... a quick check for `localStorage` there would probably resolve this.

**Comment by zgreen** [comment]
No problem; as noted, it's probably a very small fix. I'll put a PR together.

**Comment by skipjack** [comment]
Great, feel free to ping us if you have any questions re build process or anything else.

**Comment by skipjack** [comment]
Closed via #1249
